app/frontend/src/features/communities/useCommunity.ts

```diff
@@ -120,3 +126,24 @@ export const useListNearbyUsers = (communityId?: number) =>
         lastPage.nextPageToken ? lastPage.nextPageToken : undefined,
     }
   );
+
+export const useNewDiscussionMutation = (queryClient: QueryClient) =>
+  useMutation<
+    Discussion.AsObject,
+    GrpcError,
+    {
+      title: string;
+      content: string;
+      ownerCommunityId: number;
+    }
+  >(
+    ({ title, content, ownerCommunityId }) =>
+      service.discussions.createDiscussion(title, content, ownerCommunityId),
+    {
+      onSuccess(_data, { ownerCommunityId }) {
```

Comment:
```suggestion
      onSuccess(_, { ownerCommunityId }) {
```
Nit: since we aren't using `data` at all?